src/main.js

```diff
@@ -24,8 +26,13 @@ Vue.filter("length", (l) => {
 Vue.use(Vuetify);
 Vue.directive("click-outside", ClickOutside);
 Vue.use(Meta, { refreshOnceOnNavigation: true });
+Vue.use(PiniaVuePlugin);
+Vue.use(Router);
```

Comment:
Moved this from `router.js` since the ordering with `PiniaVuePlugin` is necessary. This will disappear anyway once we can move to vue 3.